lib/mailtrap/template.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+module Mailtrap
+  # Data Transfer Object for Email Template Request
+  # @attr_reader name [String] The template name (required, <= 255 chars)
+  # @attr_reader category [String] The template category (required, <= 255 chars)
+  # @attr_reader subject [String] The email subject (required, <= 255 chars)
+  # @attr_reader body_text [String] The plain text content (<= 10000000 chars)
+  # @attr_reader body_html [String] The HTML content (<= 10000000 chars)
```

Comment:
Are validations enforced? How are we going to support this logic?

Reply:
no, it just hints, like other parts of annotations, enforcing validations on client side would take responsibility from API and break flexibility of it,

Reply:
If somebody changes API validation he will likely forget to look into SDK docs. Im suggesting to leave a ref to docs